connectors/src/connectors/webcrawler/temporal/client.ts

```diff
@@ -89,3 +94,43 @@ export async function stopCrawlWebsiteWorkflow(
     return new Err(e as Error);
   }
 }
+
+export async function launchCrawlWebsiteSchedulerWorkflow(): Promise<
+  Result<string, Error>
+> {
+  const client = await getTemporalClient();
+
+  const workflowId = crawlWebsiteSchedulerWorkflowId();
+  try {
```

Comment:
It would have been nice to use the signal API here.

Reply:
Not sure I get it. This is a workflow with a crown rule, I don't think signaling makes sense here. 
We use it like that for Gdrive and it has never caused any issue.
Can you elaborate?

Reply:
My point was more that we probably don't need to kill and restart the workflow every time, do we? Looking at the current usage this approach is fine for now since we only exposes it through the CLI but if tomorrow updating settings call this function it might be better to just signal the running workflow.